src/main/kotlin/de/snickit/fluffy/Utils.kt

```diff
@@ -0,0 +1,62 @@
+package de.snickit.fluffy
+
+import net.dv8tion.jda.api.Permission
+import net.dv8tion.jda.api.entities.GuildChannel
+import net.dv8tion.jda.api.entities.Member
+import org.slf4j.LoggerFactory
+import java.time.LocalDateTime
+
+object Utils {
+
+    private val logger = LoggerFactory.getLogger(this::class.java)
+
+    private val summerStartDay: LocalDateTime = LocalDateTime.of(0, 4, 1, 0, 0)
+
+    private val  winterStartDay: LocalDateTime = LocalDateTime.of(0, 10, 1, 0, 0)
+
+    private const val gracePeriodInDays: Long = 30
+
+    private fun getSemesterForDate(date: LocalDateTime): String {
+        val summerSameYear = summerStartDay.withYear(date.year).plusDays(gracePeriodInDays)
+
+        if (date.isBefore(summerSameYear)) {
+            return "${date.year}WS"
+        }
+
+        val winterSameYear = winterStartDay.withYear(date.year).plusDays(gracePeriodInDays)
+        if (date.isBefore(winterSameYear)) {
+            return "${date.year}SS"
+        }
+
+        return "${date.year + 1}WS"
+    }
+
+    fun getCurrentSemester(): String =
+        getSemesterForDate(LocalDateTime.now())
+
+
+    fun assignCategory(categoryName: String, guildChannel: GuildChannel, sync: Boolean) {
+        val cats = guildChannel.guild.getCategoriesByName(categoryName, true)
+        if (cats.size != 1) {
+            throw Exception("Category $categoryName does not exist or is ambivalent!")
+        }
+        val parentCat = cats.first()
+        logger.info("assign category ${parentCat.name}")
+        guildChannel.manager.setParent(parentCat)
+        if(sync) {
+            logger.info("syncing permissions with ${parentCat.name}")
+            guildChannel.manager.sync(parentCat)
+        }
+    }
+
+    fun addMembersToChannel(guildChannel: GuildChannel, channelMembers: List<Member>) {
```

Comment:
```suggestion
    fun addMembersToChannel(guildChannel: GuildChannel, vararg channelMembers: Member) {
```

Reply:
Warum soll hier nicht eine Liste verwendet werden? 
Ich habe die Member in Form einer Liste vorliegen und finde es umständlich und sogar verwirrend, diese nicht als Liste, sondern als Varargs zu deklarieren

Reply:
Die Idee ist, dass man so "einfacher" einen einzelnen User hinzufügen könnte. Passt aber :D